optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/score/director/TestdataDifferentValuesCalculator.java

```diff
@@ -0,0 +1,32 @@
+package org.optaplanner.core.impl.testdata.score.director;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.optaplanner.core.api.score.buildin.simple.SimpleScore;
+import org.optaplanner.core.impl.score.director.easy.EasyScoreCalculator;
+import org.optaplanner.core.impl.testdata.domain.TestdataEntity;
+import org.optaplanner.core.impl.testdata.domain.TestdataSolution;
+import org.optaplanner.core.impl.testdata.domain.TestdataValue;
+
+public class TestdataDifferentValuesCalculator implements EasyScoreCalculator<TestdataSolution> {
```

Comment:
Personally, since the change from the corrupted to this one is just in the one line at the end, I'd merge both impls in one class and differentiate them by a boolean constructor argument or something. That way, the difference between the two calculators will be more obvious.

Reply:
Since ScoreDirectorFactoryConfig does not take an instance of ScoreCalculator, but a Class object of a Calculator instead, I am not sure this would be a plausible solution. Do you think this can be done in a different way or it should be kept as it is?

Reply:
@michaltomco You can have two classes sharing a super-class, which could have this boolean switch in the constructor.

(I'm kinda sensitive to code duplication. We can probably live without this.)